perf/benchmark/run_benchmark_job.sh

```diff
@@ -131,6 +131,19 @@ function collect_clusters_info() {
   collect_envoy_info "${1}" "${FORTIO_SERVER_POD}" "clusters"
 }
 
+function read_perf_test_conf()
+{
+  perf_test_conf="${1}"
+  while IFS="=" read -r key value; do
```

Comment:
how was this being done before?
Can we just source this file?

Reply:
source is not secure, it is not the recommended way i think.